### PR TITLE
SAIC-789 Fix broken CopyFromGlanceToGlance

### PR DESCRIPTION
**Pull request by aguzikova**
Glance image migration fails with KeyError: 'owner_name' on the Devlab.
(Regression bug caused by SAIC-760.)

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module cloudferrylib.os.image.glance_image
cloudferrylib/os/image/glance_image.py:225: [W0212(protected-access), GlanceImage.get_ref_image] Access to a protected member _resp of a client class
cloudferrylib/os/image/glance_image.py:305: [W0613(unused-argument), GlanceImage.read_info] Unused argument 'kwargs'
cloudferrylib/os/image/glance_image.py:372: [W0221(arguments-differ), GlanceImage.deploy] Arguments number differs from overridden 'deploy' method

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/1805/

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

File: cloudferrylib/os/image/glance_image.py [hunk 1]
```diff
@@ -307,8 +314,11 @@ def read_info(self, **kwargs):
         for glance_image in self.get_image_list():
             info = self.make_image_info(glance_image, info)
 
+        def get_tags():
+            return {}
```
Comment (roman-verchikov) [inline cloudferrylib/os/image/glance_image.py]: Not sure why this change is needed. Please either remove `get_tags()` method completely, or leave it as is

Comment (aguzikova) [inline cloudferrylib/os/image/glance_image.py]: Ok.

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module cloudferrylib.os.image.glance_image
cloudferrylib/os/image/glance_image.py:225: [W0212(protected-access), GlanceImage.get_ref_image] Access to a protected member _resp of a client class
cloudferrylib/os/image/glance_image.py:305: [W0613(unused-argument), GlanceImage.read_info] Unused argument 'kwargs'
cloudferrylib/os/image/glance_image.py:369: [W0221(arguments-differ), GlanceImage.deploy] Arguments number differs from overridden 'deploy' method

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module cloudferrylib.os.image.glance_image
cloudferrylib/os/image/glance_image.py:225: [W0212(protected-access), GlanceImage.get_ref_image] Access to a protected member _resp of a client class
cloudferrylib/os/image/glance_image.py:305: [W0613(unused-argument), GlanceImage.read_info] Unused argument 'kwargs'
cloudferrylib/os/image/glance_image.py:369: [W0221(arguments-differ), GlanceImage.deploy] Arguments number differs from overridden 'deploy' method

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by roman-verchikov** [comment]
+1

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module cloudferrylib.os.image.glance_image
cloudferrylib/os/image/glance_image.py:230: [W0212(protected-access), GlanceImage.get_ref_image] Access to a protected member _resp of a client class
cloudferrylib/os/image/glance_image.py:310: [W0613(unused-argument), GlanceImage.read_info] Unused argument 'kwargs'
cloudferrylib/os/image/glance_image.py:374: [W0221(arguments-differ), GlanceImage.deploy] Arguments number differs from overridden 'deploy' method

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by plomakin** [comment]
+1, works for me

**Comment by roman-verchikov** [comment]
can be merged
